### PR TITLE
chore: setup eslint

### PR DESCRIPTION
**Pull request by MarcusNotheis**

### CODE REVIEW

File: packages/main/src/components/AnalyticalTable/index.tsx [hunk 1]
```diff
@@ -1088,7 +1088,7 @@ const AnalyticalTable = forwardRef((props: AnalyticalTablePropTypes, ref: Ref<HT
         )}
       </div>
       <Text aria-hidden="true" id={`smartScaleModeHelper-${uniqueId}`} className={classes.hiddenSmartColMeasure}>
-        ""
+        {''}
```
Comment (Lukas742) [inline packages/main/src/components/AnalyticalTable/index.tsx:1091]: I think we can just self-close this `Text`, so no need for an empty string to be rendered.

File: packages/main/src/components/Form/index.tsx [hunk 2]
```diff
@@ -101,6 +101,7 @@ export interface FormPropTypes extends CommonProps {
 
 const clonedChildrenForSingleColumn = (reactChildren, currentLabelSpan) =>
   React.Children.map(reactChildren, (child) => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```
Comment (Lukas742) [inline packages/main/src/components/Form/index.tsx:104]: `@ts-ignore`, shouldn't be necessary here, I probably overlooked that in the initial implementation

**Review by Lukas742 — CHANGES_REQUESTED** [review]

File: packages/main/src/components/AnalyticalTable/index.tsx (hunk 1 above)
Comment (MarcusNotheis) [reply packages/main/src/components/AnalyticalTable/index.tsx:1091]: children are mandatory

File: packages/main/src/components/Form/index.tsx (hunk 2 above)
Comment (MarcusNotheis) [reply packages/main/src/components/Form/index.tsx]: done

**Review by Lukas742 — APPROVED** [review]

**Pull request merged**